docs/gen_reference_pages.py

```diff
@@ -0,0 +1,90 @@
+import mkdocs_gen_files
+from era5cli import inputref
+import era5cli.cli
+import subprocess
+from typing import List, Generator
+
+
+def divide_chunks(biglist: List, n: int) -> Generator[List, None, None]:
+    """Divide a list in n-sized chunks"""
+    for i in range(0, len(biglist), n):
+        yield biglist[i : i + n]
+
+
+def add_padding(multiline_string: List[str]):
+    """Add a 4-spaces padding to the start of every line."""
+    multiline_string = multiline_string.replace("\n", "\n    ")
+    return f"    {multiline_string}"
+
+
+# Build the variable reference markdown file from the input reference.
+filename = "reference/variables.md"
+
+variable_data = [  # Full name, short name, inputreference, number of columns:
+    ("Available single level variables", "2D vars", inputref.PLVARS, 3),
+    ("Available pressure level variables", "3D vars", inputref.SLVARS, 2),
+    (
+        "Available pressure levels",
+        "Pressure levels",
+        [str(p) for p in inputref.PLEVELS],
+        4,
+    ),
+    ("Available ERA5-land variables", "ERA5-Land", inputref.ERA5_LAND_VARS, 2),
+]
+
+
+filename = "reference/variables.md"  # Already exists, has the introductory text
```

Comment:
aha! That's not obvious. Perhaps use https://github.com/ryneeverett/python-markdown-comments to insert an inline comment in the file explaining that info will be added through this script?